script.js

```diff
@@ -76,4 +76,221 @@ form.addEventListener('submit', (event) => {
+    div.className = 'details1';
+    div.id = index;
+    div.innerHTML = `
+    <h2 class="card1-head">${value.heading}</h2>
+    <p class="data-arc">${value.text}</p>
+    <div class="programs">
+        <ul class="position-items">
+            <li class="stack-position">html </li>
+            <li class="stack-position">bootstrap </li>
+            <li class="stack-position">Ruby</li>
+        </ul> 
+    </div>
+    <button type="button" class="see-project-btn button-hov hov">See Project </button>
+    `;
+  } else if (index === 2 || index === 5) {
+    div.className = 'details2';
+    div.id = index;
+    div.innerHTML = `
+    <h2 class="mobile-head">${value.heading}</h2>
+    <h2 class="web-padd">${value.heading2}</h2>
+    <p class="data-arc">${value.text}</p>
+    <div class="programs">
+        <ul class="position-items">
+            <li class="stack-position">html </li>
+            <li class="stack-position">bootstrap </li>
+            <li class="stack-position">Ruby</li>
+        </ul>
+    </div>
+    <button type="button" class="see-project-btn button-hov hov">See Project </button>
+    `;
+  }
+  // Appendong work project to the div
+  work.appendChild(div);
+});
+
+const btnClick = document.querySelectorAll('.see-project-btn');
+const btnClick1 = document.querySelectorAll('.get-my-resume-btn');
+
+btnClick.forEach((btn) => {
+  btn.addEventListener('click', (e) => {
+    e.preventDefault();
+    const x = btn.parentNode.id;
+    const pop = document.createElement('div');
+    const overlay = document.getElementById('overlay');
+    pop.className = 'popUp';
+    pop.style.display = 'block';
+    overlay.style.display = 'block';
+    pop.innerHTML = `
+    <div id="top-popup">
+    <h3 class="top-stories mobile-heading">${cardData[x].heading}</h3>
+    <h3 class="top-stories desktop-heading">${cardData[x].heading}</h3>
+    <h3 class="top-stories top-hd">${cardData[x].heading3} </h3>
+<h3 class="top-hd">Keeping track of hundreds  of components website</h3>
+    <button class="close-button">&times;</button>
+    
+    <div class="programs">
+        <div class="position-buttons">
+            <button class="recentBtn" type="button" id="color">html </button>
+            <button class="recentBtn" type="button" id="color">bootstrap </button>
+            <button class="recentBtn" type="button" id="color">Ruby</button>
+        </div>
+    </div>
+   </div> 
+<div class="destop5">
+<div class="popup-photo">
+<img class="recent-pic5" src="images/snapshoot.png" alt="snapshoot Portfolio">
+</div>
+  
+    <div class="popup-para">
+    <span class="recent-para1 ">
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent </>
```

Comment:
- As requested by the previous reviewer, kindly make sure the images are showing up to match the Figma design:
![Screenshot from 2023-04-22 17-53-39](https://user-images.githubusercontent.com/97834160/233794605-84076ff7-a585-48bb-a109-7d10c43bc293.png)

Reply:
hi  [GedeonTS](https://github.com/GedeonTS)

it showing on my side I don know why I did it from the previous
![image](https://user-images.githubusercontent.com/110924992/233795835-addcec15-58f0-4f33-bb98-ea13f85e4d8d.png)